### ISSUE TITLE
Some GUI Screens are not updating backgrouns.

### ISSUE THREAD

**Issue by AzureDoom** [Keksuccino/FancyMenu#10]
**Describe the bug**
I am trying to change the background of the Video, Audio, and Controls menus but they are not updating.

**To Reproduce**
1. Create Layout for Video Settings.
2. Set an animated background
3. Save and close Layout Editor.
4. See no change in background.

**Expected behavior**
The background for the video, controls, resource packs would change. 

**Screenshots**
Here is the working OptionsScreen
![image](https://user-images.githubusercontent.com/7415711/79892164-1b23f900-83d0-11ea-8713-0e7b942039f1.png)
But the Video Settings is not using the correct background.
![image](https://user-images.githubusercontent.com/7415711/79892191-2a0aab80-83d0-11ea-83eb-5d03acc91c83.png)

**Layout**
```
type = menu

customization-meta {
  identifier = net.minecraft.client.gui.screen.VideoSettingsScreen
  renderorder = foreground
}

customization {
  name = rcraft_options
  action = animatebackground
  replayanimation = false
}
```

**Basic Informations (please complete the following information):**
 - OS: Windows
 - FancyMenu Version: 1.0.beta
 - Forge Version 31.1.46
 - Minecraft Version 1.15.2

**Comment by Keksuccino** [comment]
That happens in screens with lists. As example, all controls buttons in the controls screen are in such a "list". All lists in vanilla screens have this darker looking dirt background. It's difficult to implement all customization features for these kind of screens, but I will try it! Thanks for telling me!

**Comment by AzureDoom** [comment]
No problem! Loving the update!

**Comment by Keksuccino** [comment]
Fixed with FancyMenu v1.0.1.
